content/docs/user-guide/pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,256 @@
+# Defining Pipelines
+
+Pipelines represent data workflows that you want to **reproduce** reliably -- so
+the results are consistent. The regular pipelining process involves:
+
+- Obtain and `dvc add` or `dvc import` the project's initial data requirements
+  (see [Data Management]). This <abbr>caches</abbr> the data and generates
+  `.dvc` files.
+
+- Define the pipeline [stages](#stages) in `dvc.yaml` files (more on this
+  later). Example structure:
+
+  ```yaml
+  stages:
+    prepare: ... # stage 1 definition
+    train: ... # stage 2 definition
+    evaluate: ... # stage 3 definition
+  ```
+
+- Capture other useful metadata such as runtime
+  [parameters](#parameter-dependencies), performance [metrics], and [plots] to
+  visualize. DVC supports multiple file formats for these.
+
+<admon type="info">
+
+We call this file-based definition _codification_ (YAML format in our case). It
```

Comment:
I think it's fine to have it in use case, but no here - not practical, doesn't have any useful information

Reply:
I hear you but the point of this sentence is to get to the next one. I want to say that you can get on GitOps and that's possible via codification. Should I just throw the term "codified" somewhere in there without explaining it?

Reply:
Like

> Codifying your pipeline with DVC has the added benefit of allowing you to develop pipelines on standard Git workflows ([GitOps]).